tests/test_aos.py

```diff
@@ -76,15 +76,16 @@ def test_array_interface():
     # print('particle 2 from aos:\n',aos[1])
     # print('array interface\n', aos.__array_interface__)
     arr = aos.to_numpy()
+    int_arg = 7 if arr[0][0].dtype == "float32" else 6  # padding
```

Comment:
Is this something that users will have to know, that there might be this extra padding element so the indices for the integers might be shifted?